config/options.yml

```diff
@@ -49,7 +49,6 @@ rpm_repository:
         :python-pylxca: !ruby/regexp /.+-2\.1\.1.+/
         :python-unittest2: !ruby/regexp /.+-1\.1\.0.+/
         :python-vspk: !ruby/regexp /.+-5\.3\.2.+/
-        :qpid-proton: !ruby/regexp /.+-0\.30\.0.+/
```

Comment:
do we intend to backport this to quinteros?  (I don't think we wanted to, but just checking)

Reply:
We haven't decided if we're shipping this 3.1 now or later.  It's still TBD.